patches/llvm-project/0015-Multilib-support-for-libraries-with-exceptions.patch

```diff
@@ -0,0 +1,27 @@
+From 26fe33c4db8b786077e34966a4d9e8b40dc90745 Mon Sep 17 00:00:00 2001
+From: Piotr Przybyla <piotr.przybyla@arm.com>
+Date: Wed, 29 Nov 2023 14:05:00 +0000
+Subject: [PATCH 15/15] Multilib support for libraries with exceptions
+
+---
+ clang/lib/Driver/ToolChains/BareMetal.cpp | 4 ++++
```

Comment:
Should we fix in this upstream, indead of adding a patch?

Reply:
Yes we can do it.